copycat/conceptMapping.py

```diff
@@ -79,6 +79,7 @@ def sameTargetDescriptor(self, other):
     def sameDescriptors(self, other):
         if self.sameInitialDescriptor(other):
             return self.sameTargetDescriptor(other)
+        return False
```

Comment:
I'm no longer pushing for any of these specific diffs to be adopted into your repo, since my own fork has so many changes that it's now getting closer to a "rewrite" than a "patched version". Happy for you to adopt them; happy for you to ignore them.

FWIW on this particular diff, just to explain my reasoning: I think it's a Very Good Thing to make `sameDescriptors` return consistently `True` or `False`, because then I can describe it (in the hypothetical API docs, or verbally to a human reader, or via a static typing system such as [mypy](http://mypy-lang.org)) as "returning `bool`". If I leave it as returning "either `True` or `None`", then I have to amend all of my explanations (to humans or to machines) as "well, this function returns something that's *kind of* `bool` but it might also return `None`; but when it returns `None`, the caller needs to treat that as if it were `False`"... I mean it just gets hairier.
So that's why I added this line in my fork.